src/firefly/js/visualize/ui/PngViewer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {useStoreConnector} from '../../ui/SimpleComponent';
+import {getActiveTableId, getTblInfoById, getCellValue} from '../../tables/TableUtil';
+
+export function PngViewer() {
+    const [tbl_id] = useStoreConnector(() => () => getActiveTableId());
```

Comment:
I have not tested this yet, but I don't think this should work, If is does we might have a problem with `getStoreConnector`.  As the user clicks row to row the `tbl_id` does not change so the component as it is written should not update.  You really need go get `tbl_id` and `highlightedRow`.

Reply:
It's working, but I am guessing it's because it's rendering more than it needs to.